applications/StructuralMechanicsApplication/custom_elements/spring_damper_element_3D2N.hpp

```diff
@@ -6,14 +6,12 @@
 //  License:		 BSD License
 //					 license: structural_mechanics_application/license.txt
 //
-//  Main authors:    
+//  Main authors:    Quirin Aumman
```

Comment:
Aumann

Reply:
Ups, sorry, that is what happens when you assume you remember the thing correctly and you don't need to copy paste :D